packages/authorization-updater/src/readModelService.ts

```diff
@@ -33,7 +41,32 @@ export function readModelServiceBuilder(
     return undefined;
   }
 
+  async function getClientsPurpose(purposeId: PurposeId): Promise<Client[]> {
+    const data = await clients
+      .find(
+        {
+          "data.purposes.purpose.purposeId": purposeId,
```

Comment:
Could you double-check this? Inside `client`, the `purposes` field is an array of ids. Check [this](https://github.com/pagopa/interop-be-monorepo/pull/528/files#diff-a764f99cffde894c5ed8bc42d54e7b5b06d44750711eeff58845ad1f93a1eea7R15-R26)

Reply:
cc @galales

Reply:
This is based on the old client model